Open Judge System/Workers/OJS.Workers.Common/Extensions/ProcessExecutionResultExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace OJS.Workers.Common.Extensions
+{
+    using System;
+
+    public static class ProcessExecutionResultExtensions
+    {
+        public static void OffsetTimeWorked(this ProcessExecutionResult result, int baseTimeUsed)
+        {
+            // Display the TimeWorked, when the process was killed for being too slow (TotalProcessorTime is still usually under the timeLimit when a process is killed),
+            // otherwise display TotalProcessorTime, so people have an acurate idea of the time their program used
```

Comment:
Remove "people". Data, in this case time measurement, is not target at people only. Substitute with something like "...so that the final result is as close as possible to the actual worker time"